src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigCommand.cs

```diff
@@ -55,11 +55,36 @@ internal class ConfigCommand
                             WorkingDirectory = workingDirectory.Value,
                             ShowPath = showPath.HasValue()
                         };
-
                         ConfigGetRunner.Run(args, getLogger);
                         return 0;
                     });
                 });
+                ConfigCmd.Command("set", SetCmd =>
+                {
+                    CommandArgument configKey = SetCmd.Argument(
+                        "CONFIG_KEY",
+                        Strings.ConfigSetConfigKeyDescription);
+                    CommandArgument configValue = SetCmd.Argument(
+                        "CONFIG_VALUE",
+                        Strings.ConfigSetConfigValueDescription);
+                    var configFile = SetCmd.Option(
+                        "--configfile",
+                        Strings.ConfigSetConfigFileDescription,
```

Comment:
how about using the existing string `Option_ConfigFile` like other commands? https://github.com/search?q=repo%3ANuGet%2FNuGet.Client+Option_ConfigFile&type=code